app/schemas/com.github.dedis.student20_pop.model.data.LAODatabase/1.json

```diff
@@ -0,0 +1,320 @@
+            "name": "index_Meeting_id",
+            "unique": false,
+            "columnNames": [
+              "id"
+            ],
+            "createSql": "CREATE INDEX IF NOT EXISTS `index_Meeting_id` ON `${TABLE_NAME}` (`id`)"
+          },
+          {
+            "name": "index_Meeting_laoChannel",
+            "unique": false,
+            "columnNames": [
+              "laoChannel"
+            ],
+            "createSql": "CREATE INDEX IF NOT EXISTS `index_Meeting_laoChannel` ON `${TABLE_NAME}` (`laoChannel`)"
+          }
+        ],
+        "foreignKeys": []
+      },
+      {
+        "tableName": "LAOWitnessCrossRef",
+        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`channel` TEXT NOT NULL, `publicKey` TEXT NOT NULL, PRIMARY KEY(`channel`, `publicKey`))",
+        "fields": [
+          {
+            "fieldPath": "channel",
+            "columnName": "channel",
+            "affinity": "TEXT",
+            "notNull": true
+          },
+          {
+            "fieldPath": "publicKey",
+            "columnName": "publicKey",
+            "affinity": "TEXT",
+            "notNull": true
+          }
+        ],
+        "primaryKey": {
+          "columnNames": [
+            "channel",
+            "publicKey"
+          ],
+          "autoGenerate": false
+        },
+        "indices": [
+          {
+            "name": "index_LAOWitnessCrossRef_publicKey",
+            "unique": false,
+            "columnNames": [
+              "publicKey"
+            ],
+            "createSql": "CREATE INDEX IF NOT EXISTS `index_LAOWitnessCrossRef_publicKey` ON `${TABLE_NAME}` (`publicKey`)"
+          }
+        ],
+        "foreignKeys": []
+      },
+      {
+        "tableName": "RollCall",
+        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`id` TEXT NOT NULL, `laoChannel` TEXT, `name` TEXT, `creation` INTEGER, `start` INTEGER, `scheduled` INTEGER, `location` TEXT, `description` TEXT, PRIMARY KEY(`id`))",
+        "fields": [
+          {
+            "fieldPath": "id",
+            "columnName": "id",
+            "affinity": "TEXT",
+            "notNull": true
+          },
+          {
+            "fieldPath": "laoChannel",
+            "columnName": "laoChannel",
+            "affinity": "TEXT",
+            "notNull": false
+          },
+          {
+            "fieldPath": "name",
+            "columnName": "name",
+            "affinity": "TEXT",
+            "notNull": false
+          },
+          {
+            "fieldPath": "creation",
+            "columnName": "creation",
+            "affinity": "INTEGER",
+            "notNull": false
+          },
+          {
+            "fieldPath": "start",
+            "columnName": "start",
+            "affinity": "INTEGER",
+            "notNull": false
+          },
+          {
+            "fieldPath": "scheduled",
+            "columnName": "scheduled",
+            "affinity": "INTEGER",
+            "notNull": false
```

Comment:
When a Roll Call is scheduled it doesn't have a start time yet, and vice-versa when a roll-call starts it doesn't have a scheduled time. For this reason, if the RollCall is scheduled what value would the start element have?